mods/ts/weapons/explosions.yaml

```diff
@@ -47,27 +44,21 @@ CyborgExplode:
 		ImpactSounds: expnew10.aud
 
 TiberiumExplosion:
+	Inherits: ^DamagingExplosion
 	Warhead@1Dam: SpreadDamage
 		Spread: 9
 		Damage: 10
-		Versus:
-			None: 90
-			Wood: 75
-			Light: 60
-			Heavy: 25
 		DamageTypes: Prone50Percent, TriggerProne, ExplosionDeath
-	Warhead@2Res: CreateResource
+	Warhead@Res: CreateResource
 		AddsResourceType: Tiberium
 		Size: 1,1
-	Warhead@3Eff: CreateEffect
+	Warhead@2Eff: CreateEffect
 		Explosions: large_explosion
-		ExplosionPalette: effectalpha75
-		ImpactSounds: expnew09.aud
+	-Warhead@4Smu: LeaveSmudge
 
 SmallDebris:
 	ReloadDelay: 60
 	Range: 4c0
```

Comment:
2 Questions:
1. Why isn't `SmallDebris` getting a similar inheritance like `^DamagingExplosion` and `UnitExplode`? I mean something like:
```yaml
^Debris:
    ReloadDelay: 60
    AndSoOn: ...

SmallDebris:
    Inhertis: ^Debris

LargeDebris:
    Inherits: ^Debris
    YadaYada: ...

```

2. Why got `- Report:` removed (I'm quite sure there is valid reason, I trying to find possible differences)?